src/HtmlParser.js

```diff
@@ -93,17 +97,30 @@ export default class HtmlParser {
       if (detect.hasOwnProperty(type)) {
         if (detect[type].test(this.stream)) {
           const token = streamReaders[type](this.stream);
+
           if (token) {
-            token.text = this.stream.substr(0, token.length);
-            return token;
+            if (token.type == 'startTag' &&
+                (/script|style/i).test(token.tagName)) {
```

Comment:
This was actually more than these two https://github.com/krux/postscribe/blob/1.4.0/htmlParser/htmlParser.js#L76